### PR TITLE
Fixes errors in the README

### PR DESCRIPTION
**Pull request by AmauryCarrade**
As requested ;)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
 
-For example,
+As example,
 
 ```
 https://doha.blueslime.fr/api/check/29b2b527-1b59-45df-b7b0-d5ab20d8731a
 ```
 
-respond,
+answers,
```
Comment (AmauryCarrade) [inline README.md:25]: A note on respectful usage or rate limiting would be useful, maybe.

File: README.md [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
-We are providing you a RESTful API who simply take an UUID and respond you if this
-UUID exists in the database or not.
+We provide a RESTful API, taking an UUID and answering if this UUID exists in the database or not.
 
 ```
 https://doha.blueslime.fr/api/check/<UUID>
 ```
 
-For example,
+As example,
```
Comment (marysaka) [inline README.md:19]: Well it's basicly the same

**Review by marysaka — APPROVED** [review]

**Comment by AmauryCarrade** [comment]
Sounds better IMO

File: README.md (hunk 2 above)
Comment (AmauryCarrade) [reply README.md:19]: Sounds better IMO

**Pull request merged**